metadata-etl/src/main/resources/jython/ElasticSearchIndex.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, args):
     self.logger = LoggerFactory.getLogger('jython script : ' + self.__class__.__name__)
     self.elasticsearch_index_url = args[Constant.WH_ELASTICSEARCH_URL_KEY]
     self.elasticsearch_port = args[Constant.WH_ELASTICSEARCH_PORT_KEY]
+    self.elasticsearch_index = args[Constant.WH_ELASTICSEARCH_INDEX_KEY]
```

Comment:
Not anywhere in the code. or should we set it as "wherehows" if nothing  in args[Constant.WH_ELASTICSEARCH_INDEX_KEY] is found?
Looks like We do not have such checks on other parameters yet